DigitalLearningSolutions.Web/Controllers/NotificationPreferencesController.cs

```diff
@@ -61,11 +68,17 @@ public IActionResult UpdateNotificationPreferences(string userType)
         [Authorize]
         [HttpPost]
         [Route("/NotificationPreferences/Edit/{userType}")]
-        public IActionResult SaveNotificationPreferences(string userType, IEnumerable<int> notificationIds)
+        public IActionResult SaveNotificationPreferences(UserType userType, IEnumerable<int> notificationIds)

```

Comment:
Can we have a `UserType?` instead actually? That makes the nullability much more obvious to readers.
Same on other endpoint.

Reply:
Does it work to stick a `?` on `UserType` here?